### PR TITLE
GEOMESA-3045 Include wps jar in gs-plugins

### PR DESCRIPTION
**Pull request by elahrvivaz**
Signed-off-by: Emilio Lahr-Vivaz <elahrvivaz@ccri.com>

### CODE REVIEW

**Comment by elahrvivaz** [comment]
I verified that the dist and gs plugins still have the same jars, except for the one wps jar being included now.

File: geomesa-archetypes/geomesa-archetypes-gs-plugin/pom.xml [hunk 1]
```diff
@@ -174,6 +174,23 @@
         </dependencies>
     </dependencyManagement>
 
+    <dependencies>
+        <dependency>
+            <groupId>org.locationtech.geomesa</groupId>
+            <artifactId>geomesa-process-wps_${scala.binary.version}</artifactId>
```
Comment (jnh5y) [inline geomesa-archetypes/geomesa-archetypes-gs-plugin/pom.xml:180]: Where does this inclusion add the jar?  Is it part of the gs-plugin tar.gz?

**Comment by jnh5y** [comment]
Should this PR impact lines 18 or 29 here: https://github.com/locationtech/geomesa/blob/main/geomesa-archetypes/geomesa-archetypes-binary-dist/assembly.xml#L18-L29 ?

File: geomesa-archetypes/geomesa-archetypes-gs-plugin/pom.xml (hunk 1 above)
Comment (elahrvivaz) [reply geomesa-archetypes/geomesa-archetypes-gs-plugin/pom.xml:180]: right, it makes it compile scope and then it gets picked up by the dependencies section of the maven assembly plugin

**Comment by elahrvivaz** [comment]
> Should this PR impact lines 18 or 29 here: https://github.com/locationtech/geomesa/blob/main/geomesa-archetypes/geomesa-archetypes-binary-dist/assembly.xml#L18-L29 ?

no, i left it in the gs-plugins folder in the binary dist for backwards compatibility (we can remove it later), and we don't need/want it in the tools themselves since it only goes in geoserver. and the gs-plugin tar.gz gets pulled in separately so it's not affected by the includes/excludes

**Pull request merged**